lib/KettleRequest.js

```diff
@@ -267,8 +268,8 @@ kettle.request.http.successHandler = function (request, response) {
 };
 
 /**
- * Ensure that the response is properly disposed of when finished.
- * @param {Object} that request object.
+ * Ensure that the response is properly disposed of when finished
+ * @param {Component} The `kettle.request` component
```

Comment:
The parameter name got trimmed on the last edit:

> /Users/duhrer/Source/rtf/kettle.amb26/lib/KettleRequest.js
  270:1  error  Expected JSDoc for 'that' but found 'The'  valid-jsdoc